src/bosh-director/lib/bosh/director/addon/addon_filter.rb

```diff
@@ -48,7 +51,7 @@ def self.verify_jobs_section(applicable_jobs, filter_type)
           name = safe_property(job, 'name', :class => String, :default => '')
           release = safe_property(job, 'release', :class => String, :default => '')
           if name.empty? || release.empty?
-            raise RuntimeIncompleteFilterJobSection.new("Job #{job} in runtime config's #{filter_type} section must " +
+            raise IncompleteFilterJobSection.new("Job #{job} in config's #{filter_type} section must " +
```

Comment:
Should we be explicit about where the bad addon was located to avoid the scenario where a runtime config has an error, but the user is getting an error about something they can't find in their deployment manifest.